### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by valfirst** [serenity-bdd/serenity-jbehave#260]
Hi @wakaleo !

The last commit on `master` was 2 years ago. The number of issues in this repository is growing and it seems the project isn't actively maintained.

Is this project still alive?

**Comment by wakaleo** [comment]
No, JBehave is no longer supported in Serenity BDD. We advice moving to Cucumber instead.
